yazi-core/Cargo.toml

```diff
@@ -12,9 +12,9 @@ repository  = "https://github.com/sxyazi/yazi"
 yazi-adaptor   = { path = "../yazi-adaptor", version = "0.2.5" }
 yazi-boot      = { path = "../yazi-boot", version = "0.2.5" }
 yazi-config    = { path = "../yazi-config", version = "0.2.5" }
-yazi-dds       = { path = "../yazi-dds", version = "0.2.5" }
-yazi-plugin    = { path = "../yazi-plugin", version = "0.2.5" }
-yazi-proxy     = { path = "../yazi-proxy", version = "0.2.5" }
+yazi-dds       = { path = "../yazi-dds", version = "0.2.5", default-features = false }
```

Comment:
Why do we need to add `default-features = false` to `yazi-core`? `yazi-core` is not the entry point of the program, is this necessary?

Reply:
Yes, because that dependency would otherwise expect to have those features, and we cannot override that without making enabling that feature not the default.

Reply:
`cargo tree` seems unreliable for these tests, so I just `eza -l target/release` and compare the sizes. Without this there is no size difference between the default and `--no-default-features`, but with this there is a difference.

Reply:
What command did you use for building?

Reply:
`cargo build` with and without `--release` and with and without `--no-default-features`